docs/source/user/api.rst

```diff
@@ -52,6 +56,21 @@ or in the form of a query string parameter as shown below.
 
 Requests which contain an invalid token will receive a ``403`` HTTP error.
 
+To use Radius Token authentication method `send a request using user API to obtain the auth token <#login-obtain-user-auth-token>`_. This can be integrated in captive portal to be send on user login.
+The token is valid for authentication only as long as freeradius accounting ``Stop`` request
+is not send or token is not deleted. Once the token is created, the following can be used to send the request:
```

Comment:
since this is the recommended way, we can move it before the old method

Reply:
Done! :smile: